src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/DeprecationControl.xaml

```diff
@@ -0,0 +1,76 @@
+<UserControl x:Class="NuGet.PackageManagement.UI.DeprecationControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:nuget="clr-namespace:NuGet.PackageManagement.UI"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             mc:Ignorable="d"
+             d:DesignHeight="150" d:DesignWidth="350">
+  <UserControl.Resources>
+    <ResourceDictionary>
+      <ResourceDictionary.MergedDictionaries>
+        <nuget:SharedResources />
+      </ResourceDictionary.MergedDictionaries>
+      <BitmapImage
+        x:Key="BitmapImage_StatusWarning"
+        UriSource="/NuGet.PackageManagement.UI;component/Resources/statuswarning.16.16.png" />
+    </ResourceDictionary>
+  </UserControl.Resources>
+  <Grid>
+    <Expander
+      Margin="0,12,0,16"
+      IsExpanded="True"
+      Foreground="{DynamicResource {x:Static nuget:Brushes.UIText}}">
+      <Expander.Header>
+        <StackPanel
+          Orientation="Horizontal"
+          VerticalAlignment="Center">
+          <TextBlock
+            FontWeight="Bold"
+            Text="{x:Static nuget:Resources.Label_Deprecated}"/>
+          <Image
```

Comment:
Use CrispImage as in https://github.com/NuGet/NuGet.Client/blob/15da55e34d7f0504c94516e42840d133f0fb60ba/src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/LicenseAcceptanceWindow.xaml#L66-L73

We probably don't even need the status warning resource anymore do we? 
That probably should be removed.

Reply:
Fixed with https://github.com/NuGet/NuGet.Client/pull/2899/commits/2962ec86f20b56a13e431d9061189b2e89de7c85 and https://github.com/NuGet/NuGet.Client/pull/2899/commits/6805bd0c82a90db23334292a57c18ce95f1fbe6a